docs/04-documentation/LG-04-09.adoc

```diff
@@ -6,8 +6,8 @@
 Softwarearchitekt:innen kennen:
 
 * Grundlagen mehrerer publizierter Frameworks zur Beschreibung von Softwarearchitekturen, beispielsweise:
-** ISO/IEC/IEEE 42010, siehe <<iso42010>>
-** arc42, siehe <<arc42>>
+** ISO/IEC/IEEE 42010, 
+** arc42,
 ** C4, siehe <<brownc4>>
 ** FMC, siehe <<fmc>>
```

Comment:
same as in LG 04-03: why are brownc4 and fmc still referenced?

Reply:
see above.